packages/asc-ui/src/components/Breadcrumbs/Breadcrumbs.test.tsx

```diff
@@ -0,0 +1,27 @@
+import { render } from '@testing-library/react'
+import React from 'react'
+import Breadcrumbs from './Breadcrumbs'
+import BreadcrumbsItem from './BreadcrumbsItem/BreadcrumbsItem'
+
+describe('Breadcrumbs', () => {
+  it('should render', () => {
+    const { container, queryByText } = render(
+      <Breadcrumbs>
+        <BreadcrumbsItem href="/">Home</BreadcrumbsItem>
+        <BreadcrumbsItem href="/innovatie">Innovatie</BreadcrumbsItem>
+        <BreadcrumbsItem href="/innovatie/bouwprojecten">
+          Bouw- en verkkeersprojecten
+        </BreadcrumbsItem>
+        <BreadcrumbsItem href="/innovatie/bouwprojecten/contact">
+          Contact
+        </BreadcrumbsItem>
+      </Breadcrumbs>,
+    )
+
+    expect(container.firstChild).toBeDefined()
+
+    expect(queryByText('Home')).toBeInTheDocument()
+    expect(queryByText('Innovatie')).toBeInTheDocument()
```

Comment:
To be complete, you could add `expect(queryByText('Bouw- en verkkeersprojecten')).toBeInTheDocument()` as well

Reply:
Or add a snapshot to test the whole thing including links and icons.

Reply:
ik dacht dat we met snapshots zouden stoppen: ik was die laatste vergeten

Reply:
Zou kunnen hoor, weet ik niet.

Reply:
ik heb hem toegevoegd: i zie verder geen toMatchSnapshot in de source code